src/main/kotlin/com/cout970/magneticraft/block/BlockCoke.kt

```diff
@@ -0,0 +1,19 @@
+package com.cout970.magneticraft.block
+
+import coffee.cypher.mcextlib.extensions.blocks.creativeTab
+import com.cout970.magneticraft.fuel.FuelProvider
+import com.cout970.magneticraft.item.ItemCoke
+import com.cout970.magneticraft.misc.CreativeTabMg
+import net.minecraft.block.Block
+import net.minecraft.block.material.Material
+
+object BlockCoke : BlockBase(
+        material = Material.ROCK,
+        registryName = "block_coke"), FuelProvider<Block> {
+
+    init {
+        creativeTab = CreativeTabMg
```

Comment:
The creative tab is specified in the super-class BlockBase, there is no need to duplicate it.